include/cell.h

```diff
@@ -222,6 +224,29 @@ class Cell {
                                     unsigned phase, double pvolume,
                                     const Eigen::Matrix<double, 6, 1>& pstress);
 
+  //! Assign velocity constraint
+  //! Directions can take values between 0 and Dim-1
+  //! \param[in] face_id Face of cell of velocity constraint
+  //! \param[in] dir Direction of velocity constraint
+  //! \param[in] velocity Applied velocity constraint
+  bool assign_cell_velocity_constraint(unsigned face_id, unsigned dir,
+                                       double velocity);
+
+  //! Apply velocity constraints
+  void apply_cell_velocity_constraints();
+
+  //! Compute normal vector
+  void compute_normals();
+
+  //! Return number of normal vectors
+  //! \retval number of normal vectors
+  unsigned nnormal() const { return face_normals_.size(); }
```

Comment:
Shouldn't this be equal to the number of faces a cell has?

Reply:
No, better to only calculate normal of the face with boundary conditions

Reply:
OK and done!